cpp/src/reductions/sum_of_squares.cu

```diff
@@ -33,9 +33,10 @@ std::unique_ptr<cudf::scalar> sum_of_squares(column_view const& col,
     simple::detail::element_type_dispatcher<cudf::reduction::op::sum_of_squares>{},
     col,
     output_dtype,
+    std::nullopt,
     stream,
     mr);
 }
 
 }  // namespace reduction
-}  // namespace cudf
+}  // namespace cudf
```

Comment:
We should keep one empty line at the end of each file. This is an implicit convention.